src/styles/style.css

```diff
@@ -0,0 +1,5 @@
+@import './blocks/stars.css';
+
+body {
+  margin: 0;
```

Comment:
Consider adding a fallback font family to enhance cross-browser compatibility. For example, if you're using a sans-serif font, your CSS could look like this:
body {
  margin: 0;
  font-family: 'YourFont', sans-serif;
}
This ensures that if the primary font fails to load for any reason, the browser will use a default sans-serif font instead.